include/uv.h

```diff
@@ -1099,6 +1099,11 @@ typedef struct {
   long tv_usec;
 } uv_timeval_t;
 
+typedef struct {
+  int64_t tv_sec;
+  int64_t tv_usec;
```

Comment:
`tv_usec` could be `uint32_t`. There generally aren't more than one million microseconds in a second. :-)

It might make sense to upgrade it to nanoseconds.

Reply:
Alignment ?
Anyway `nsec` it is